src/test/regress/pg_regress_multi.pl

```diff
@@ -251,6 +251,20 @@ END
     close $fh;
 }
 
+sub write_settings_to_postgres_conf
+{
+    my ($pgOptions, $pgConfigPath) = @_;
+    print $pgConfigPath;
```

Comment:
The output of this is a bit weird and maybe not so useful:
```
tmp_check/master/data/postgresql.confwaiting for server to start.... done
server started
tmp_check/worker.57637/data/postgresql.confwaiting for server to start.... done
server started
tmp_check/worker.57638/data/postgresql.confwaiting for server to start.... done
server started
CREATE DATABASE
CREATE EXTENSION
CREATE FUNCTION

```

Reply:
this is a left over, removing